Assignment4/ErHulio-z170088/task06.py

```diff
@@ -0,0 +1,65 @@
+# %% [markdown]
+# **Task 06: Modifying RDF(s)**
+
+# %%
+#pip install rdflib 
+github_storage = "https://raw.githubusercontent.com/FacultadInformatica-LinkedData/Curso2021-2022/master/Assignment4/course_materials"
+
+# %% [markdown]
+# Leemos el fichero RDF de la forma que lo hemos venido haciendo
+
+# %%
+from rdflib import Graph, Namespace, Literal
+from rdflib.namespace import RDF, RDFS
+g = Graph()
+g.namespace_manager.bind('ns', Namespace("http://somewhere#"), override=False)
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/resources/example5.rdf", format="xml")
+
+# %% [markdown]
+# Create a new class named Researcher
+
+# %%
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+# %% [markdown]
+# **TASK 6.1: Create a new class named "University"**
+
+# %%
+g.add((ns.University, RDF.type, RDFS.Class))
+
+# %% [markdown]
+# **TASK 6.2: Add "Researcher" as a subclass of "Person"**
+
+# %%
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+
+# %% [markdown]
+# **TASK 6.3: Create a new individual of Researcher named "Jane Smith"**
+
+# %%
+g.add((ns.JaneSmith, RDF.type, ns.Researcher))
+
+# %% [markdown]
+# **TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**
+
+# %%
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.JaneSmith, VCARD.FN, Literal("Jane Smith")))
+g.add((ns.JaneSmith, VCARD.Given, Literal("Jane")))
+g.add((ns.JaneSmith, VCARD.Family, Literal("Smith")))
+
+# %% [markdown]
+# **TASK 6.5: Add UPM as the university where John Smith works**
+
+# %%
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((ns.JohnSmith, VCARD.worksAt, ns.UPM))
```

Comment:
vcard does not definde the property worksAt. You have to search for another vocabulary or define your own property